client/src/sections/learning-paths/create/InitialSection.tsx

```diff
@@ -31,10 +32,14 @@ interface InitialSectionProps {
 const InitialSection = ({ register, errors, control }: InitialSectionProps): JSX.Element => {
   const { values: learningPath, editMode } = useAppSelector((state) => state.learningPath);
   const dispatch = useAppDispatch();
-  const categoriesOption: MultiSelectOptionData[] = categories.map(({ id, name }) => ({
-    value: id,
-    label: name,
-  }));
+
+  const { data: categories = [] } = useGetCategoryQuery(null);
```

Comment:
Please define the type in the query, so we will not need to pass `null` or `undefined` to get the category, it would be just `useGetCategoryQuery()`